src/tpt_reports/report_generator.py

```diff
@@ -170,8 +170,6 @@ def format_table(df, header_style, column_widths, column_style_list):
     return table
 
 
-# Issue #26 - test report_gen()
-# TODO: Add unit tests for following logic and remove this comment.
```

Comment:
Are these comments being removed in this PR because they weren't removed in #56 where `test_report_gen()` was first created?

Reply:
That works and is no problem.  When I've done something similar in the past, I like to mention it in the PR description for full transparency and to reduce confusion during reviews.